### PR TITLE
Delete the live messages of offline users

### PR DESCRIPTION
**Pull request by Hepic**

### CODE REVIEW

**Comment by gtklocker** [comment]
You should mention this change in etc/spec/ARCHITECTURE.md.

File: client/message/history.jsx [hunk 1]
```diff
@@ -31,6 +31,11 @@ const History = React.createClass({
             myUsername: null
         };
     },
+    deleteLiveMessage(deletedMessageid)
+    {
```
Comment (gtklocker) [inline client/message/history.jsx]: nit: style

File: realtime/server.js [hunk 2]
```diff
@@ -195,7 +195,7 @@ socket.on('connection', function (client) {
 
         delete people[client.id];
         delete usernames[username];
-        socket.sockets.emit('part', username);
+        socket.sockets.emit('part', {username: username, deletedMessageid: deletedMessageid});
```
Comment (gtklocker) [inline realtime/server.js]: nit: style, when your object contains more than 2 keys expand it in multiple lines like
```
{
    username: username,
    password: password,
}
```

**Comment by dionyziz** [comment]
User-facing changes must also be mentioned in SPECIFICATION

**Comment by gtklocker** [comment]
@dionyziz It seems like the Jest tests [are getting killed](https://travis-ci.org/dionyziz/ting/builds/92565445#L7827) causing the build to fail, can you take a look?

**Comment by dionyziz** [comment]
Not sure why the tests are getting killed. Do you think this is a timeout
issue?

**Pull request closed without merging**